app/fullService/api/buildGiftCode.ts

```diff
@@ -58,11 +58,15 @@ const buildGiftCode = async ({
   const { txProposal, giftCode, giftCodeB58 } = result;
 
   // TODO fix type, right now it just matches what the component is expecting
-  const totalValueConfirmation = txProposal.outlayList
-    .map((outlay) => BigInt(outlay.value))
-    .reduce((acc, cur) => acc + cur, BigInt(0));
+  // const totalValueConfirmation = txProposal.outlayList
```

Comment:
Ignore changes in gift code stuff for now. Gift codes aren't supported in FS api v2 yet